transports/sqs/configuration-options_policy_sqs_[5.3,).partial.md

```diff
@@ -2,53 +2,53 @@
 
 include: configuration-options-policy-intro
 
-The transport creates one policy statement with a combined condition for all events it subscribes to. An example policy for the events described above would look as follows:
+The transport creates a policy statement for the event types it subscribes to:
 
 ```json
 {
-    "Version" : "2012-10-17",
-    "Statement" : [
-        {
-            ...
-            "Action"    : "sqs:SendMessage",
-            "Resource"  : "arn:aws:sqs:some-region:someaccount:endpoint",
-            "Condition" : {
-                "ArnLike" : {
-                    "aws:SourceArn" : [
-                        "arn:aws:sns:some-region:someaccount:Sales-OrderAccepted",
-                        "arn:aws:sns:some-region:someaccount:Sales-OrderPaid"
-                    ]
-                }
-            }
-        },
-    ]
+  ...
+  "Statement": [
+    {
+      ...
+      "Action": "sqs:SendMessage",
+      "Resource": "arn:aws:sqs:some-region:some-account:endpoint",
+      "Condition": {
+        "ArnLike": {
+          "aws:SourceArn": [
+            "arn:aws:sns:some-region:some-account:Sales-OrderAccepted",
+            "arn:aws:sns:some-region:some-account:Sales-OrderPaid"
+          ]
+        }
+      }
+    }
+  ]
 }
 ```
 
-The combined condition is extended when an endpoint manually subscribes to an event type by calling [`session.Subscribe<CustomEvent>()`](/nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md). Unsubscribing does not modify the policy on the endpoint's input queue.
+The policy statement is updated when an endpoint explicitly subscribes to an event type using [`session.Subscribe<CustomEvent>()`](/nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md). Unsubscribing does not modify the policy.
 
 ### Wildcards
 
-#### Acount condition
+#### Account condition
 
-Allow all messages from any topic in the account to pass. No value needs to be provided, the account name will be derived from the subscribed topic ARN.
+Allow all messages from any topic in the account. The account name is derived from the subscribed topic ARN.
```

Comment:
Is it actually _derived_ from the ARN (the ARN is passed through an algorithm)? Or is more _extracted_ (it's a substring in the ARN)?

Reply:
Extracted via a substring. Does that matter? Isn't derived good enough?

Reply:
It's minor, but I believe extracted is more accurate, so I'll change it to that and merge.